app/src/server/api/routers/fineTunes.router.ts

```diff
@@ -180,6 +182,23 @@ export const fineTunesRouter = createTRPCRouter({
         input.baseModel.baseModel,
       );
 
+      await kysely
```

Comment:
How long does this operation take with a 100k row dataset? I understand why you moved it here and it definitely simplifies the code, but I'm worried we might end up providing a pretty bad UX if this takes 20+ seconds with large datasets.

Eventually I think we probably will have to move it back to the background, since I expect we'll end up supporting 1M+ row datasets. But don't have to optimize for that use case yet.

Reply:
I’m not sure, let me test how long that takes. I do think that in most cases creating the training entries as part of the initial fine-tune creation handler results in a better UX. This way when you navigate to the fine-tunes page, training entries is not initially. set to 0, which is confusing and feels buggy. I would much rather wait 20 seconds and know that it worked.

Reply:
Creating a fine-tune with about 80k training rows (for a dataset with 100k rows in total) took about 8 seconds locally. It didn't feel like a long wait, and I was happy to immediately see the training set size when I navigated to the fine-tunes page.